### PR TITLE
mds: fix usage() display 

### PR DESCRIPTION
**Pull request by renhwztetecs**
The usage using derr, will display the date and time;
remove unnecessary break.

Signed-off-by: renhwztetecs <ren.huanwen@zte.com.cn>

### CODE REVIEW

File: src/ceph_mds.cc [hunk 1]
```diff
@@ -108,7 +108,6 @@ int main(int argc, const char **argv)
     }
     else if (ceph_argparse_flag(args, i, "--help", "-h", (char*)NULL)) {
       usage();
-      break;
```
Comment (jcsp) [inline src/ceph_mds.cc]: It's not obvious when reading that usage() will ultimately call exit(), so a comment would be nice here

**Comment by jcsp** [comment]
+1 for the fix to the usage part, Reviewed-By

Would be nice to add a comment where the break was removed

**Comment by renhwztetecs** [comment]
@jcsp 
thanks for your review
I  add a comment where the break was removed

**Comment by jcsp** [comment]
Tested here http://pulpito.ceph.com/jspray-2016-06-05_19:36:13-fs-wip-jcsp-testing-20160603---basic-mira/

**Pull request merged**